M2/Macaulay2/m2/code.m2

```diff
@@ -271,6 +262,33 @@ debuggerHook = entering -> (
 	  )
      )
 
+-- disassemble
+CodeList := new Type of List
```

Comment:
Oooh you've opened a can of worms (to be dealt with in future, not this PR) .. now I want `code` to return a `CodeList` with numbered corresponding to the line of code!